BICC.R

```diff
@@ -0,0 +1,26 @@
+BICC <- function(x, w, p, Alpha=NULL, Beta=NULL, Theta=0.8)
```

Comment:
Please, exclude this file from the commit, because it seems to be a copy of the correctly placed \R\BICC.R (below).